### DISCUSSION TITLE
Empty .pcss file work not build.

### DISCUSSION THREAD

**Discussion by ghost** [postcss/postcss#1629] [Q&A]
I did @import an empty .pcss file.

The file in .css is incomplete.

Am I not supposed to @import an empty file?

Do you need an example?

**Comment by ai** [comment]
> The file in .css is incomplete.

Show PostCSS config, input CSS files (try to reduce the size) and expected/real output.

**Reply by ghost** [reply]
The same error is not appearing.

I'll test it again and continue the conversation.
